src/platform/iotjs_module_gpio-linux-general.inl.h

```diff
@@ -260,7 +265,8 @@ bool SetPinDirection(int32_t pin, GpioDirection direction) {
     strcpy(buffer, "out");
   }
 
-  DDDLOG("GPIO SetPinDirection() - path: %s, dir: %s", direction_path, buffer);
+  DDDLOG("GPIO SetPinDirection() - path: %s, dir: %s\n",
+      direction_path, buffer);
```

Comment:
Please indent this line according to the [Coding style guideline](https://github.com/Samsung/iotjs/wiki/Coding%20Style%20Guideline#function-call)